### PR TITLE
add missing windowdesktop token type

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/setup/admin/adding-nodes.mdx [hunk 1]
```diff
@@ -40,7 +40,7 @@ Auth Service. To receive a host certificate upon joining a cluster, a new
 Teleport host must present an **invite token**.
 
 An invite token also defines which role a new host can assume within a cluster:
-`auth`, `proxy`, `node`, `app`, `kube`, or `db`.
+`auth`, `proxy`, `node`, `app`, `kube`, `db`, or `windowsdesktop`.
```
Comment (zmb3) [inline docs/pages/setup/admin/adding-nodes.mdx:43]: To be honest I'm not sure this makes sense.

We generally use the term "node" to mean an instance running the Teleport SSH server, accessed via `tsh` or the web UI. This page is about adding nodes.

You don't need to know about Windows (or db, kube, etc) tokens in order to learn how to add a node to a cluster. This extra information may confuse a reader more than it helps.

@ptgott what do you think?

Comment (stevenGravy) [reply docs/pages/setup/admin/adding-nodes.mdx:43]: Ok, I think a better thing is to update this cli which doesn't have all the token types.  Will close this and update there

image

**Comment by stevenGravy** [comment]
closing this and will update the cli reference to include all token types

File: docs/pages/setup/admin/adding-nodes.mdx (hunk 1 above)
Comment (ptgott) [reply docs/pages/setup/admin/adding-nodes.mdx:43]: I like the idea of having an all-purpose guide to joining a Teleport instance to a cluster regardless of role, and this guide is mostly there. 

After we settle #10976, I think we should edit this page to use that new term (e.g., "agent") as soon as possible. In the meantime, we already mention non-Node services in this sentence, so I think it's okay to use `windowsdesktop` here.

**Pull request closed without merging**